Fix PivotGrid - calculateCustomSummary issues (#1269)
DevExpress/devextreme-documentation: api-reference/30 Data Layer/PivotGridDataSource/1 Configuration/fields/calculateCustomSummary.md

The total summary value.

##### field(options.value): any

> A value to be aggregated. If you need to access all properties of each cell value, do not specify the [dataField](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#dataField) option for the field that contains this function.

Reviewer: ```suggestion
A value to be aggregated. If you need to access an entire data object in this field, do not specify the [dataField](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#dataField) option.
```

---

> A summary value calculation is conducted in three stages: *start*, in which the **totalValue** is initialized, *calculate*, in which it is modified, and *finalize*, in which the **totalValue** is adjusted. To identify the current stage, check the value of the **summaryProcess** field that belongs to the function's parameter.

Reviewer 2: ```suggestion
A summary value calculation is conducted in three stages: *start* - in which the **totalValue** is initialized, *calculate* - in which it is modified, and *finalize* - in which the **totalValue** is adjusted. To identify the current stage, check the value of the **summaryProcess** field that belongs to the function's parameter.
```

---

> [note]If you need to access all properties of each cell value, do not specify the [dataField](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#dataField) option for the field that contains this function.

Reviewer: I think the `options.value` description is a more suitable place for this information.
